readme.md

```diff
@@ -74,6 +74,11 @@ depending on context.
     [`Italian Profanity` from WikiPedia][it-source]
```

Comment:
I sorted the languages, so now Italian is here 👆🏼 not 👇🏽

Seems the linter is barking at the Markdown syntax, could you fix that? I can also do that if it’s too much effort.

Reply:
Sorry for the late answer, these days were dedicated to high priority tasks.
Thanks for the fixes!

Reply:
No problem!